command/agent/node_pool_endpoint.go

```diff
@@ -125,6 +128,10 @@ func (s *HTTPServer) nodePoolDelete(resp http.ResponseWriter, req *http.Request,
 }
 
 func (s *HTTPServer) nodePoolNodesList(resp http.ResponseWriter, req *http.Request, poolName string) (interface{}, error) {
+	if req.Method != http.MethodGet {
+		return nil, CodedError(http.StatusMethodNotAllowed, ErrInvalidMethod)
+	}
```

Comment:
@lgfa29 when I rebased on `main` I added this assertion so we don't list on `POST` or whatever.

Reply:
Ah good catch, thanks!